src/migrations/20240306000000-monitoring-nonbulkcreate.js

```diff
@@ -0,0 +1,17 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      // await prepMigration(queryInterface, transaction, sessionSig);
+      // eslint-disable-next-line @typescript-eslint/quotes
+      await queryInterface.sequelize.query(/* sql */`SELECT setval('"ImportFiles_id_seq"', COALESCE((SELECT MAX(id)+1 FROM "ImportFiles"), 1), false);`, { transaction });
+      // eslint-disable-next-line @typescript-eslint/quotes
+      await queryInterface.sequelize.query(/* sql */`SELECT setval('"ImportDataFiles_id_seq"', COALESCE((SELECT MAX(id)+1 FROM "ImportDataFiles"), 1), false);`, { transaction });
+    });
+  },
+};
```

Comment:
Can we also include a down migration?

Reply:
the migration is just the fix the issue with the current state where the value in the id column and the sequence do not align. there is nothing to do in a down

Reply:
An empty down migration is fine.

Reply:
If we don't include anything and need to perform a rollback, the SequelizeMeta table would become out of sync.

Reply:
ok i will put in an empty down